setup.py

```diff
@@ -1,14 +1,35 @@
 import os
 import warnings
 from packaging import version
```

Comment:
This will still fail under some circumstances. Correct?

Why not simply `try:` to `import packaging` and skip the checks when necessary.

Reply:
After merging #70 we will only need `version` for comparing versions w.r.t some warnings. I think we can easily use `try` here and if it fails issue a warning `some warnings are not shown, since the module packaging could not be found on your system`.

I can take care of implementing this here.

Reply:
done in 73451b8.